backend/apps/accounts/views.py

```
@@ -515,509 +120,46 @@ def post(self, request):
         )
 
 
-# ─────────────────────────────────────────────────────────────────────────────
-# OTP (Email Verification) Views
-# ─────────────────────────────────────────────────────────────────────────────
-
-
 @extend_schema(
-    request=OtpRequestSerializer,
-    responses=OpenApiResponse(description="OTP sent to email if account exists."),
+    responses=OpenApiResponse(description="Check if reset token is valid."),
```

Ruff (0.15.20)

[error] Undefined name `OpenApiResponse`

(F821)